auspice/server/getDataset.js

```diff
@@ -58,6 +72,9 @@ const getDataset = async (req, res) => {
   if (!query.prefix) {
     return helpers.handleError(res, `getDataset request must define a prefix`);
   }
+
+  redirectInrbDrc(res, query);
```

Comment:
If a redirect is sent in this function call, then `getDataset` should return instead of continuing to process the request. The simplest way to do that in this case is going to be to inline the function into the callsite and add a `return` statement if the redirect needs to happen.

You can see the evidence of the continued request processing in the server logs. For example, requesting `/getDataset?prefix=/inrb-drc/ebola/2019-12-09` results in logs for both paths:

```
Getting (nextstrain) datasets for: prefix=/inrb-drc/ebola/2019-12-09
[verbose]       Guessing tree name -- this should be improved
[verbose]       Dataset:  { fetchUrls:
   { main: 'http://data.nextstrain.org/inrb-drc_ebola_2019-12-09.json',
     tree:
      'http://data.nextstrain.org/inrb-drc_ebola_2019-12-09_tree.json',
     meta:
      'http://data.nextstrain.org/inrb-drc_ebola_2019-12-09_meta.json' },
  auspiceDisplayUrl: 'inrb-drc/ebola/2019-12-09',
  treeName: undefined,
  secondTreeName: undefined,
  source: CoreSource {} }
[verbose]       Fetching http://data.nextstrain.org/inrb-drc_ebola_2019-12-09.json

Getting (nextstrain) datasets for: prefix=/groups/inrb-drc/ebola/2019-12-09
[verbose]       Can't compare against available datasets as there are none!
[verbose]       Guessing tree name -- this should be improved
[verbose]       Dataset:  { fetchUrls:
   { main:
      'https://nextstrain-inrb.s3.amazonaws.com/ebola_2019-12-09.json?…',
     tree:
      'https://nextstrain-inrb.s3.amazonaws.com/ebola_2019-12-09_tree.json?…',
     meta:
      'https://nextstrain-inrb.s3.amazonaws.com/ebola_2019-12-09_meta.json?…' },
  auspiceDisplayUrl: 'groups/inrb-drc/ebola/2019-12-09',
  treeName: undefined,
  secondTreeName: undefined,
  source: InrbDrcSource {} }
[verbose]       Fetching https://nextstrain-inrb.s3.amazonaws.com/ebola_2019-12-09.json?…
```

Reply:
How are you seeing verbose logging output? I've tried setting a few `DEBUG` type variables to verbose as well as using a `--verbose` flag on `npm run start` (similar to what we use for auspice), but to no avail.

Reply:
@kairstenfay `node server.js --verbose` or `npm run start -- --verbose`